src/hotspot/share/opto/library_call.cpp

```diff
@@ -5923,68 +5923,17 @@ bool LibraryCallKit::inline_multiplyToLen() {
   // Set the original stack and the reexecute bit for the interpreter to reexecute
   // the bytecode that invokes BigInteger.multiplyToLen() if deoptimization happens
   // on the return from z array allocation in runtime.
```

Comment:
Since we are not allocating z array during runtime anymore, do we still need these comments?

Reply:
Thanks for pointing it out! Removed.